iroh-rpc-client/src/store.rs

```diff
@@ -83,4 +84,15 @@ impl StoreClient {
         let size = self.backend.get_size(req).await?.size;
         Ok(size)
     }
+
+    #[tracing::instrument(skip(self))]
+    pub async fn get_block_cids(&self) -> Result<impl Stream<Item = Result<Cid>>> {
```

Comment:
Heads up, with tarpc we don't have streaming builtin so we will need to do something more manual here

Reply:
Have we decided already that we do tarpc? I guess I could do my own paging by exposing the id to the interface.